packages/components/src/Checkbox/Checkbox.test.tsx

```diff
@@ -39,6 +40,37 @@ it("renders each variation of checked, defaultChecked and indeterminate", () =>
   });
 });
 
+it("renders a description when set", () => {
+  const { getByText } = render(<Checkbox description="Checkers" />);
+  const description = getByText("Checkers");
+  expect(description).toBeInstanceOf(HTMLParagraphElement);
+});
+
+it("allows react components to be passed as children", () => {
+  const handleLinkClick = jest.fn();
+  const { getByText } = render(
+    <Checkbox description="Checkers">
+      <Text>
+        I agree to the
+        <a
+          style={{
```

Comment:
just wondering why did you have style code here? tests should not care about the style, and even if it cares, we shouldn't define a new style in the test

Reply:
I understand we should have `<a>` here, but I was just thinking we don't necessarily need to have it here. what do you think about that?

Reply:
Yes, that makes sense to me. The test should work without it